aws/sdk/examples/kinesis/src/bin/put-record.rs

```diff
@@ -3,83 +3,88 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-use std::process;
-
-use kinesis::{Client, Config, Region};
-
-use aws_types::region::{EnvironmentProvider, ProvideRegion};
-
+use aws_types::region::ProvideRegion;
+use kinesis::{Client, Config, Error, Region, PKG_VERSION};
 use structopt::StructOpt;
-use tracing_subscriber::fmt::format::FmtSpan;
-use tracing_subscriber::fmt::SubscriberBuilder;
 
 #[derive(Debug, StructOpt)]
 struct Opt {
-    /// The region
+    /// The default AWS Region.
     #[structopt(short, long)]
-    region: Option<String>,
+    default_region: Option<String>,
```

Comment:
I'm not sure I would call this `default_region` since it actually overrides any other region settings. Why not `region`?

Reply:
Yes. I should do so everywhere, but I'll wait until I need to change an example before I do so.